itest/lnd_test.go

```diff
@@ -183,7 +184,17 @@ func getTestCaseSplitTranche() ([]*lntest.TestCase, uint, uint) {
 		trancheEnd = numCases
 	}
 
-	return allTestCases[trancheOffset:trancheEnd], threadID,
+	// Shuffle the test cases so the blocks generated in each tranche are
+	// more likely to be even.
+	shuffledTestCases := make([]*lntest.TestCase, len(allTestCases))
+	copy(shuffledTestCases, allTestCases)
+	rand.Seed(time.Now().UnixNano())
+	rand.Shuffle(len(shuffledTestCases), func(i, j int) {
+		shuffledTestCases[i], shuffledTestCases[j] =
+			shuffledTestCases[j], shuffledTestCases[i]
+	})
+
+	return shuffledTestCases[trancheOffset:trancheEnd], threadID,
```

Comment:
Unfortunately, we can't do this... Because the 4 tranches are created by a shell script, each tranche will be its own process and shuffle things differently. So we'll end up not covering some tests...

How about we analyze the run time of some of the tests and shuffle them manually to get a better spread?

Reply:
ahhhh shoot...yeah this won't work🤦🏻

> shuffle them manually

You mean modify the `list_on.go`?

Reply:
Yes, manually tweak the order in `list_on.go` to move around some of the items we know take a long time.

Reply:
cool looks like this trick save us roughly 15min
```
run ubuntu itests (btcd, backend=btcd)  35m 53s
```